api/tools/src/cron/export-to-rdr.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+curl -X GET 'http://localhost:8081/v1/cron/exportToRdr' \
```

Comment:
Now that we can test locally with a local task executor, this was slightly helpful to have.